### PR TITLE
Adds two new recipes for Death Spice

### PR DESCRIPTION
**Pull request by MintyPhresh**
[Chemistry][feature]{add to wiki]
## About the PR 
Adds two new recipes for death spice, one for chemlabs and one for hobochem/clever botanical splices:
(at < -80C) 
(2) Charcoal + (1) Neurotoxin -> (3) Death Spice

(any temperature)
(1) Paper + (1) Pepper + (0.4) Nicotine + (1) Space Fungus -> (3) Death Spice

## Why's this needed? 
Death spice is a silly and "harmless" chem that's terribly rare on station! We could use a little more in our lives.

## Changelog 

```changelog
(u)Mintyphresh
(+)Deathspice is now craftable:
(-80C) (2) Charcoal + (1) Neurotoxin -> (3) Death Spice
(1) Paper + (1) Pepper + (0.4) Nicotine + (1) Space Fungus -> (3) Death Spice
```

### CODE REVIEW

**Comment by Valtsu0** [comment]
(2) Charcoal + (1) Neurotoxin -> (3) Death Spice

While this halves the efficiency of charcoal against ling stings it does make it instant (only for classic)
One could carry a 100u charcoal pill to instantly remove a ling sting

**Comment by MintyPhresh** [comment]
thank you-- that wasn't the intent. Will push a fix today

**Comment by MintyPhresh** [comment]
Would adding a minimum temperature be a suitable fix? Ling Island Iced Tea does the same thing with its neurotoxin reaction

File: code/modules/chemistry/Chemistry-Recipes.dm [hunk 1]
```diff
@@ -4335,6 +4335,23 @@
...
+	deathspice	// Clean chemlab version. Requiring cooling after hot rxns should make bulk production interesting
+		name = "Death Spice 1"
+		id = "death_spice 1"
+		result = "death_spice"
+		max_temperature = T0C - 80
+		result_amount = 3
+		required_reagents = list("charcoal" = 2, "neurotoxin" = 1)
+		mix_phrase = "The mixture shimmers, drying into a fine, iridescent powder."
+		mix_sound = 'sound/misc/fuse.ogg'
+
```
Comment (Tarmunora) [inline code/modules/chemistry/Chemistry-Recipes.dm:4347]: I'd drop this recipe entirely imo

**Comment by Tarmunora** [comment]
on second thought, after a brief consultation in admin chat, Death Spice is something that's better if kept rare and weird - more of an easter egg than anything else.

**Pull request closed without merging**